playwright-e2e/dsm/pages/miscellaneous-pages/user-and-permissions-page.ts

```diff
@@ -76,7 +76,7 @@ export default class UserPermissionPage {
 
     //Verify the email is as expected
     const studyAdminEmail = await studyAdmin.innerText();
-    expect(studyAdminEmail).toBe(email);
+    expect(studyAdminEmail).toBe(email.toLowerCase());
```

Comment:
to be certain, `expect(studyAdminEmail.toLowerCase()).toBe(email.toLowerCase());`.